02-Appetisers/Sea_Ice_Coordinates.ipynb

```diff
@@ -55,2294 +55,8 @@
    "outputs": [
```

Comment:
<p>Let's move the remark about only loading the last 10 years further down where we slice in time?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/643/discussion/#comment-2508507775'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>